client/extensions/wp-super-cache/components/navigation/index.jsx

```diff
@@ -86,12 +84,20 @@ const Navigation = ( { activeTab, site, translate } ) => {
 
 Navigation.propTypes = {
 	activeTab: PropTypes.string,
-	site: PropTypes.object,
+	siteId: PropTypes.number,
+	// connected props
+	siteSlug: PropTypes.string,
 	translate: PropTypes.func.isRequired,
 };
 
 Navigation.defaultProps = {
 	activeTab: '',
 };
 
-export default localize( Navigation );
+const connectComponent = connect(
+	( state, { siteId } ) => ( {
+		siteSlug: getSiteSlug( state, siteId )
```

Comment:
My only question is - do we actually need to pass down `siteId` from the controller at all ?
It looks like it's only used here, and we could use `getSelectedSiteSiteSlug`. Even then, there is also `getSelectedSiteId`.
That's the approach I went for in Zoninator.